packages/react-day-picker/src/components/Day/hooks/useDay.tsx

```diff
@@ -20,7 +20,7 @@ import { useDayFocus } from './useDayFocus';
 import { SelectMultipleContextValue } from '../../../contexts/SelectMultiple/SelectMultipleContext';
 import { SelectRangeContextValue } from '../../../contexts/SelectRange/SelectRangeContext';
 import { SelectSingleContextValue } from '../../../contexts/SelectSingle/SelectSingleContext';
-import { useModifiers } from '../../../hooks/useModifiers';
+import { useDayModifiers } from '../../../hooks/useDayModifiers';
```

Comment:
I wish we could use @contexts, @hooks… how can we ?

Reply:
Wouldn't webpack aliases work?

Reply:
Nevermind, we're using rollup...